### DISCUSSION TITLE
Invalid portman config due to globals

### DISCUSSION THREAD

**Discussion by keulenIT** [apideck-libraries/portman#516] [Q&A]
Hi guys, I have set up the following config file that I call through my CLI options file:

> version: 1
> tests:
>   contractTests:
>     $ref: "./definitions/contract-tests.yaml"
>   contentTests:
>     $ref: "./definitions/content-tests.yaml"
>   variationTests:
>     $ref: "./definitions/variation-tests.yaml"
> globals:
>   $ref: "./definitions/globals.yaml"

And I am getting the following error

` Invalid Portman Config:                ./portman-config.default.yaml
[
  {
    message: "'globals' property is not expected to be here",
    path: '{base}.globals',
    context: { errorType: 'additionalProperties' }
  }
]`

What is the reason for this globals error? As far as I can see, this aligns with the examples that were given except for that I define my own globals file. Is this not allowed?

**Comment by thim81** [comment]
hi @keulenIT 

Could you share the whole file?
Since it must be something specific to YAML file.
